bioblend/_tests/TestGalaxyHistories.py

```diff
@@ -70,6 +70,15 @@ def test_get_histories(self):
         all_histories = self.gi.histories.get_histories()
         assert len(all_histories) > 0
 
+        # Test limit and offset
+        first = self.gi.histories.get_histories(limit=1)
+        others = self.gi.histories.get_histories(offset=1)
+        assert len(others) > 0  # guess the test makes more sense with at least 2 histories
```

Comment:
Better not to make tests dependent on the order in which they are executed.

```suggestion
        assert len(first) == 1
```